src/librustc_codegen_ironox/context.rs

```diff
@@ -150,27 +194,7 @@ impl MiscMethods<'tcx> for CodegenCx<'ll, 'tcx> {
     }
 
     fn eh_personality(&self) -> Value {
```

Comment:
This function name made me smile ;)

Reply:
Ha, I wish it was my idea! It's part of the `MiscMethods` trait.

Reply:
Ha, I wish it was my idea! It's part of the `MiscMethods` trait.